.circleci/src/jobs/check.yml

```diff
@@ -1,36 +1,31 @@
-# Run checks and tests for native.
+# Run checks and tests.
```

Comment:
Combined tests and checks for web and native in one file to avoid running things twice.